tests/utils/mailing_test.py

```diff
@@ -1861,49 +1745,220 @@ def test_accepts_list_of_strings_as_to(self, app):
+                    'Name': 'John Doe'
+                }
+            ],
+            'Vars':
+                {
+                    'nom_structure': 'John Doe',
+                    'lien_validation_mail': f'{app_origin_url}/inscription/validation/{user.validationToken}'
+                }
+        }
+
+        # Then
+        assert email == expected
+
+    def test_make_pro_user_waiting_for_validation_by_admin_email(self, app):
+        # Given
+        user = create_user(email="test@email.com")
+        user.generate_validation_token()
+        offerer = create_offerer(name='Bar des amis')
+
+        # When
+        email = make_pro_user_waiting_for_validation_by_admin_email(user, offerer)
+        expected = {
+            'FromEmail': 'dev@passculture.app',
+            'FromName': 'pass Culture pro',
+            'Subject': f'[pass Culture pro] Votre structure {offerer.name} est en cours de validation',
+            'MJ-TemplateID': '778329',
+            'MJ-TemplateLanguage': 'true',
+            'Recipients': [
+                {
+                    'Email': 'test@email.com',
+                    'Name': 'John Doe'
+                }
+            ],
+            'Vars':
+                {
+                    'nom_structure': 'Bar des amis'
+                }
+        }
+
+        # Then
+        assert email == expected
+
+
+class MakeResetPasswordEmailWithMailjetTemplateTest:
+    @clean_database
+    @patch('utils.mailing.ENV', 'testing')
+    @patch('utils.mailing.IS_PROD', False)
+    @patch('utils.mailing.SUPPORT_EMAIL_ADDRESS', 'john.doe@support.com')
+    @patch('utils.mailing.feature_send_mail_to_users_enabled')
+    def test_should_write_email_with_right_data_when_not_prod_environment(self, feature_send_mail_to_users_enabled,
+                                                                          app):
+        # Given
+        feature_send_mail_to_users_enabled.return_value = False
+        user = create_user(email="test@email.com", first_name="Johnny", reset_password_token='ABCDEFG')
+        PcObject.save(user)
+
+        # When
+        email = make_reset_password_email_with_mailjet_template(user=user)
+
+        expected = {
+            'FromEmail': 'dev@passculture.app',
+            'FromName': 'Pass Culture',
+            'Subject': 'Réinitialisation de votre mot de passe',
+            'MJ-TemplateID': '912168',
+            'MJ-TemplateLanguage': 'true',
+            'To': 'dev@passculture.app',
+            'Vars':
+                {
+                    'prenom_user': 'Johnny',
+                    'token': 'ABCDEFG',
+                    'env': '-testing'
+                }
+        }
+
+        # Then
+        assert email == expected
+
+    @clean_database
+    @patch('utils.mailing.ENV', 'production')
+    @patch('utils.mailing.IS_PROD', True)
+    @patch('utils.mailing.SUPPORT_EMAIL_ADDRESS', 'john.doe@support.com')
+    @patch('utils.mailing.feature_send_mail_to_users_enabled')
```

Comment:
```suggestion
    @patch('utils.mailing.feature_send_mail_to_users_enabled', return_value=True)
```
Comme ça, tu peux effacer le paramètre dans le fonction et la ligne où tu initialisais.

Reply:
👍